composer.json

```diff
@@ -13,6 +13,9 @@
         "rector/rector": "~0.12",
         "webflo/drupal-finder": "^1.2"
     },
+    "conflict": {
+        "rector/rector": ">0.12.17"
+    },
```

Comment:
next follow up is to bump this to 0.12.18 and fix https://www.drupal.org/project/rector/issues/3271102